src/components/TaskItem.vue

```diff
@@ -10,15 +10,20 @@ const store = useStore()
   <div class="task prevent-select" :class="{ reminder: task.reminder }" @dblclick="store.toggleReminder(task)">
     <h3>
       {{ task.text }}
-      <i class="fas fa-times" @click="store.deleteTask(task.id)" />
+      <i-mdi-close-circle class="fas" @click="store.deleteTask(task.id)" />
     </h3>
     <p>{{ task.day }}</p>
   </div>
 </template>
 
 <style scoped>
 .fas {
+  --icon-size: 20px;
   color: red;
+  min-width: var(--icon-size);
+  min-height: var(--icon-size);
+  max-width: var(--icon-size);
+  max-height: var(--icon-size);
```

Comment:
what does it look like now?
could you, please, post a screenshot with it?

Reply:
Sure.
![image](https://user-images.githubusercontent.com/1153904/186244517-fe15d1db-a0f8-40b7-a1fd-509690a0cf16.png)

Without explicitly specifying min- and max-size these icons for some reason became smaller in todos with long titles, like this:
![image](https://user-images.githubusercontent.com/1153904/186245320-ee4b3e40-61a8-4eb5-a448-c1c176219022.png)

Reply:
👍